src/astUtils/creators.ts

```diff
@@ -74,6 +74,8 @@ const tokenDefaults = {
     [TokenKind.Semicolon]: ';',
     [TokenKind.SourceFilePathLiteral]: 'SOURCE_FILE_PATH',
     [TokenKind.SourceFunctionNameLiteral]: 'SOURCE_FUNCTION_NAME',
+    [TokenKind.SourceNamespaceRootLiteral]: 'SOURCE_NAMESPACE_ROOT',
+    [TokenKind.SourceNamespaceLiteral]: 'SOURCE_NAMESPACE',
```

Comment:
I think I want these to have the word "name" in them, to align with `SOURCE_FUNCTION_NAME`. So
```suggestion
    [TokenKind.SourceNamespaceRootNameLiteral]: 'SOURCE_NAMESPACE_ROOT_NAME',
    [TokenKind.SourceNamespaceNameLiteral]: 'SOURCE_NAMESPACE_NAME',
```

Thoughts?

Reply:
no problem with that; you're right it aligns with the existing names closer!

Reply:
open to a Slack huddle too, if desired

Reply:
This looks good to me. :)